packages/devtools/devtools-extension/vite.config.ts

```diff
@@ -0,0 +1,111 @@
+//
+// Copyright 2022 DXOS.org
+//
+
+import ReactPlugin from '@vitejs/plugin-react';
+import { resolve } from 'node:path';
+import { defineConfig } from 'vite';
+import { VitePluginFonts } from 'vite-plugin-fonts';
+import { crx as chromeExtensionPlugin } from '@crxjs/vite-plugin';
+
+import { ConfigPlugin } from '@dxos/config/vite-plugin';
+import { ThemePlugin } from '@dxos/react-components/plugin';
+import { kaiThemeExtension } from '@dxos/kai/theme-extensions';
+import { osThemeExtension } from '@dxos/react-ui/theme-extensions';
+
+import manifest from './manifest.json';
+
+const env = (value?: string) => (value ? `"${value}"` : undefined);
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  base: '', // Ensures relative path to assets.
+  optimizeDeps: {
```

Comment:
optimize deps can be omitted because it's only relevant for the dev server which I don't think is being used here?

Reply:
Yeah, you are right